preinst

```diff
@@ -24,7 +24,7 @@ if [ "$PACKAGEUPGRADE" = "true" ] ; then
     fi
 
     # Stop server to prevent db plugin / irods server mismatch
-    HIDE_OUTPUT=`su - ${IRODS_SERVICE_ACCOUNT_NAME:?} -c "python3 -c 'from scripts.irods.controller import IrodsController; IrodsController().stop()'"`
+    HIDE_OUTPUT=`su - ${IRODS_SERVICE_ACCOUNT_NAME:?} -c "cd /var/lib/irods; python3 -c 'from scripts.irods.controller import IrodsController; IrodsController().stop()'"`
```

Comment:
this is interesting, since the su with hyphen should drop the shell into the home directory of the named user...

should we use a variable here for the home directory too?

Reply:
The directory is referenced in full in other parts of the script, so I'm doing the same here.

Reply:
> this is interesting, since the su with hyphen should drop the shell into the home directory of the named user...

That only works if the user actually exists. If irods has not been setup prior to being upgraded, it results in the script printing an error during package upgrade.

Reply:
i see.   resolving.